backend/apps/owasp/models/chapter.py

```diff
@@ -61,12 +59,6 @@ def __str__(self):
         """Chapter human readable representation."""
         return f"{self.name or self.key}"
 
-    @staticmethod
-    @lru_cache
-    def active_chapters_count():
-        """Return active chapters count."""
-        return Chapter.active_chapters.count()
```

Comment:
The issues refactoring approach needs to be kept with other models too.

Reply:
ohh i didn't notice sorry